Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -2593,3 +2596,15 @@ void CLuaEngineDefs::EnginePreloadWorldArea(CVector position, std::optional<Prel
     if (option == PreloadAreaOption::ALL || option == PreloadAreaOption::COLLISIONS)
         g_pGame->GetStreaming()->LoadSceneCollision(&position);
 }
+
+bool CLuaEngineDefs::EngineRestreamModel(const std::uint16_t modelId)
+{
+    return g_pClientGame->RestreamModel(modelId);
+}
+
+bool CLuaEngineDefs::EngineRestream(std::optional<RestreamOption> option)
+{
+    g_pClientGame->Restream(option);
+
+    return true;
+}
```

Comment:
There’s no point in making the function return bool if it always returns the same value.
The function should be void, so it won’t return anything (just nil).